apps/docs/src/app/examples/cards/page.mdx

```diff
@@ -0,0 +1,18 @@
+import { CodeBlock } from "../../../components/code/CodeBlock";
+import big from "./Big?raw";
+import bigCheck from "./BigCheck?raw";
+import bigRadio from "./BigRadio?raw";
+import small from "./Small?raw";
+import smallCheck from "./SmallCheck?raw";
+import smallRadio from "./SmallRadio?raw";
+
+<div className="overflow-hidden mt-20">
+  <div className="w-101% grid  divide-border [grid-template-columns:repeat(auto-fit,minmax(480px,1fr))] -mt-px -mb-px [&>*]:p-20px">
+    <CodeBlock code={{ main: small }} layout="popup" />
+    <CodeBlock code={{ main: big }} layout="popup" />
+    <CodeBlock code={{ main: smallRadio }} layout="popup" />
+    <CodeBlock code={{ main: bigRadio }} layout="popup" />
+    <CodeBlock code={{ main: smallCheck }} layout="popup" />
+    <CodeBlock code={{ main: bigCheck }} layout="popup" />
```

Comment:
The sample are too similar to each other - this is basically the same sample 6 times!

We're also doing selection, but each sample only has 1 Card, so we can't actually see the radio/checkboxes "working"... Can't we instead have 1-2 examples with multiple cards instead?